src/components/FolderCards/FolderCards.jsx

```diff
@@ -0,0 +1,48 @@
+import styles from "./FolderCards.module.scss";
+import classNames from "classnames/bind";
+import { MainSearchBar, LinkItems, FolderToolBar, Cards } from "../";
+
+const cx = classNames.bind(styles);
+
+export const FolderCards = ({
+  links,
+  folders,
+  folderName,
+  setFolderName,
+  folderId,
+  setFolderId,
+}) => {
+  let hasCards = false;
```

Comment:
useState값이 아닌, let으로 사용하신 이유가 있으실까요 ?

Reply:
useState를 사용했을 때 제가 카드를 구현한 방식이 folderId === item.folder_id 일 때마다 계속 true로 변경하게 만들어서 아래와 같은 경고가 발생했습니다.
해결을 못해서 let으로 해두었는데 카드 구현 방식을 다시 생각해 봐야 할 것 같습니다.

Too many re-renders. React limits the number of renders to prevent an infinite loop.